tensorboard/backend/event_processing/data_ingester.py

```diff
@@ -196,3 +201,75 @@ def _parse_event_files_spec(logdir_spec):
             path = os.path.realpath(os.path.expanduser(path))
         files[path] = run_name
     return files
+
+
+def _get_filesystem_scheme(path):
+    """Extracts filesystem from a given path.
+
+    Args:
+        path: A strings representing an input log directory.
+    Returns:
+        Filesystem scheme, None if the path doesn't contain one. The filesystem
+        scheme is usually separated by `://` from the local filesystem path if
```

Comment:
Totally optional but I'd tend to put a longer comment like this up above Args (and after the 1-liner docstring), vs hiding it inside Returns.

Reply:
Fixed.